core/core.ex

```diff
@@ -105,7 +105,10 @@ defmodule AntikytheraCore do
       AntikytheraCore.TenantExecutorPoolsManager,
       AntikytheraCore.TmpdirTracker             ,
     ]
+    children_for_dev = if Antikythera.Env.runtime_env() == :prod, do: [], else: [
+      AntikytheraCore.ReductionLogWriter        ,
```

Comment:
There is no need to align with a list of a different variable.
```suggestion
      AntikytheraCore.ReductionLogWriter,
```